internal/app/skuba/node/bootstrap.go

```diff
@@ -40,8 +41,15 @@ func NewBootstrapCmd() *cobra.Command {
 		Use:   "bootstrap <node-name>",
 		Short: "Bootstraps the first master node of the cluster",
 		Run: func(cmd *cobra.Command, nodenames []string) {
+			if err := validate.NodeName(nodenames[0]); err != nil {
+				klog.Fatal(err)
+			}
+
 			bootstrapConfiguration := deployments.BootstrapConfiguration{
-				KubeadmExtraArgs: map[string]string{"ignore-preflight-errors": bootstrapOptions.ignorePreflightErrors},
+				KubeadmExtraArgs: map[string]string{
+					"ignore-preflight-errors": bootstrapOptions.ignorePreflightErrors,
+					"node-name":               nodenames[0],
```

Comment:
This is already performed here: https://github.com/SUSE/skuba/blob/e0de7024acc781dc2086a51dfeab01b033f85e82/internal/pkg/skuba/node/node.go#L34-L36, no need to set it as extra args here